packages/cmf/src/sagas/http.js

```diff
@@ -144,7 +144,7 @@ export function httpFetch(url, config, method, payload) {
 
 	const params = merge(
 		{
-			credentials: 'same-origin',
+			credentials: 'include',
```

Comment:
Need to send the credentials (cookies) on cross-origin request. From `iam.*.cloud.talend.com` to  to `app.*.cloud.talend.com`.
See https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials